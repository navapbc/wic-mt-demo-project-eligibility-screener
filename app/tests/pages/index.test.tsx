import Index from '@pages/index'

import { setup } from '../helpers/setup'
import { testAccessibility, testSnapshot } from '../helpers/sharedTests'

/**
 * Test setup
 */

const route = '/'

/**
 * Begin tests
 */

it('should match full page snapshot', () => {
  setup(route)
  testSnapshot(<Index />)
})

it('should pass accessibility scan', async () => {
  setup(route)
  await testAccessibility(<Index />)
})
