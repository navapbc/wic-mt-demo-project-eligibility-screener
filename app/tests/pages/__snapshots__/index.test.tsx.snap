// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render the heading 1`] = `
<h1>
  Start an application for WIC
</h1>
`;
